packages/web/components/Modal/IframeModal/styles.js

```diff
@@ -0,0 +1,48 @@
+import styled, { css } from 'styled-components';
+import { MdClose } from 'react-icons/md';
+
+import { Loader } from '../../Loading/styles';
+
+export const Container = styled.div`
+	${({ theme: { colors, screens } }) => css`
+		display: flex;
+		align-items: center;
+		justify-content: center;
+
+		width: 80%;
+		height: 80%;
+		background: ${colors.whiteSmoke};
+		position: relative;
+
+		iframe {
+			width: 100%;
+			height: 100%;
+			z-index: 2;
```

Comment:
What if we made our iframes responsive to follow the 16:9 ratio?

It could be done by tweaking some styles:
```
export const Container = styled.div`
	${({ theme: { colors } }) => css`
		display: flex;
		justify-content: center;
		align-items: center;

		width: 70%;
		height: 0;
		padding-bottom: 39.4%;
		background: ${colors.whiteSmoke};
		position: relative;

		iframe {
			position: absolute;
			top: 0;
			left: 0;
			width: 100%;
			height: 100%;
			z-index: 2;
		}

		> div {
			top: 45%;
			left: 46%;
		}
	`}
`;
```

Here's how it would look like:

https://user-images.githubusercontent.com/10481028/105549333-9740c980-5cdf-11eb-8530-b1dfb8e81311.mp4

All of this is just to prevent those black borders around the video.

What do you say?

Reply:
Nice idea, now there's an `aspectRatio` prop in iframe modal to allow the use of widescreen/default